insights/parsers/pcs_config.py

```diff
@@ -66,6 +66,8 @@
     ['cluster-infrastructure: corosync', 'cluster-name: cluster-1', 'dc-version: 1.1.13-10.el7_2.4-44eb2dd', 'have-watchdog: false', 'no-quorum-policy: ignore', 'stonith-enable: true', 'stonith-enabled: false']
     >>> pcs_config.get("Colocation Constraints")
     ['clone-1 with clone-x (score:INFINITY) (id:clone-INFINITY)', 'clone-2 with clone-x (score:INFINITY) (id:clone-INFINITY)']
+    >>> 'have-watchdog' in pcs_config.cluster_props_dict
```

Comment:
nitpick: since the value of `have-watchdog` is `false` in the sample output, would it be better to change this example line to show the actual value instead of checking if it's `contain`ed?

```python
>>> pcs_config.cluster_props_dict['have-watchdog']
'false'
```